packages/overlay/src/OverlayTrigger.ts

```diff
@@ -72,6 +72,7 @@ export class OverlayTrigger extends LitElement {
             <div
                 id="trigger"
                 @click=${this.onTrigger}
+                @keypress=${this.onKeypress}
```

Comment:
We can't guarantee that a user uses an interactive element below here, so I've added `keypress` with some gating, just in case. _Could_ be over kill...

Reply:
This does feel like overkill to me. I'd rather just see the documentation cover this.

In general I think we should document and have examples of best practices, and not try to "fix" bad practices.

Reply:
This makes sense to me. I'll work on the right way to skip this rule here and add a sentence or two to the documentation for this.